### PR TITLE
39 m3 create a UI to get a url that work on the analyzer

### LINKED ISSUE #39: M3: create a UI to get a url that work on the analyzer.
To provide an initial experience for users who are not familiar with Kibana, we need to create a web-based GUI that allows users to generate a video URL from an input following the video endpoint structure. The generated URL can be used by testers or developers in the player for analysis purposes. The URL ID must be auto-generated with a uuid4.

AC: 
- Create a web-based GUI that allows users to enter an input and generate a URL.
- Ensure the URL ID is auto-generated with a uuid4.

### PR DESCRIPTION
**Pull request by mcamardon**

### CODE REVIEW

File: analyzer-dashboard/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React</title>
```
Comment (JoaquinBCh) [inline analyzer-dashboard/index.html]:
```suggestion
    <title>Analyzer Dashboard</title>
```

File: analyzer-dashboard/src/components/DisplayedURL/DisplayedURL.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import './DisplayedURL.css';
+
+function DisplayedURL({ generatedURL }) {
+    return (
+        <div className="url-section">
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/DisplayedURL/DisplayedURL.jsx]: I could not be able to found url-section style, I think that we can remove the div.

File: analyzer-dashboard/src/components/InputField/InputFields.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.custom-input-group-text {
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/InputField/InputFields.css:1]: I think that this style doesn't do any change. Maybe can be deleted.

File: analyzer-dashboard/src/components/InputField/InputFields.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+import InputGroup from 'react-bootstrap/InputGroup';
+import './InputFields.css'
+
+function InputField({ value, onChange, placeholder, name, error, message }) {
+  return (
+    <InputGroup className="mb-3" htmlFor="formBasic">
+        <InputGroup.Text className="custom-input-group-text" id="basic-addon1" >{name}</InputGroup.Text>
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/InputField/InputFields.jsx]: Erase the className b(ecause is not applied) and the id.

File: analyzer-dashboard/src/App.css [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+.card {
+  padding: 2em;
+}
+
+.read-the-docs {
+  color: #888;
+}
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/App.css]: Delete this

**Review by JoaquinBCh — CHANGES_REQUESTED** [review]

**Review by coatesjuan — APPROVED** [review]

File: analyzer-dashboard/src/components/DisplayedURL/DisplayedURL.css [hunk 6]
```diff
@@ -0,0 +1,12 @@
+.url-text {
+  color: white;
+}
+
+.url-container {
+  max-height: 200px;
+  overflow: auto; 
+}
+
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/DisplayedURL/DisplayedURL.css]:
```suggestion

```

File: analyzer-dashboard/src/components/DisplayedURL/DisplayedURL.jsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import './DisplayedURL.css';
+
+function DisplayedURL({ generatedURL }) {
+    return (
+        <div className="card">
+            <div className="card-body">
+                <p>{generatedURL}</p>
+            </div>
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/DisplayedURL/DisplayedURL.jsx]:
```suggestion
            <p>{generatedURL}</p>
```

File: analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  const handleRemoveField = (index) => {
+    const newFields = [...objList];
+    newFields.splice(index, 1);
+    setobjList(newFields);
+  };
+  
+  return (
+    <div className="app">
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx]:
```suggestion
    <div>
```

File: analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+      <img src="https://montevideotech.dev/wp-content/uploads/2020/09/mvd-tech-1.png" alt="Logo" className="logo" />
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx]: This should not be here.

File: analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+      
+      <div className="container-box">
+        <div className='container'>
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx]: Leave only one ```<div>```

File: analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx [hunk 11]
```diff
@@ -0,0 +1,196 @@
...
+              <div className='col'>
+                <InputField
+                  value={field1}
+                  onChange={handleField1Change}
+                  placeholder='VideoURL: '
+                  name="VideoURL: "
+                  error={field1Error}
+                  message='Please provide a valid video URL.'
+                />
+                <span></span>
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/URLGenerator/URLGenerator.jsx]: Delete every ```<span></span>```

File: analyzer-dashboard/src/components/URLGenerator/URLGenerator.css [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  .close-icon {
+    color: #4dbaac;
+  }
+
+  .custom-button {
+    background-color: #4dbaac;
+    color: white;
+  }
```
Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/URLGenerator/URLGenerator.css]: Duplicated

Comment (JoaquinBCh) [inline analyzer-dashboard/src/components/URLGenerator/URLGenerator.css]:
```suggestion
```

**Review by JoaquinBCh — CHANGES_REQUESTED** [review]

**Review by JoaquinBCh — APPROVED** [review]

**Pull request merged**